ui/packages/catalog-ui-search/src/main/webapp/component/list-interactions/list-interactions.view.js

```diff
@@ -85,4 +114,10 @@ module.exports = Marionette.ItemView.extend({
   triggerClick() {
     this.$el.trigger('closeDropdown.' + CustomElements.getNamespace())
   },
+  downloadAsFile(res) {
```

Comment:
Might be able to use [saveFile](https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/react-component/utils/save-file/save-file.tsx#L14).